modules/hcp-get-ocp-channel.adoc

```diff
@@ -2,134 +2,74 @@
 //
 // * hosted_control_planes/hcp-updating.adoc
 
-:_mod-docs-content-type: CONCEPT
+:_mod-docs-content-type: PROCEDURE
 [id="hcp-get-ocp-channel_{context}"]
-= Setting channels in a hosted cluster
+= Updating your hosted cluster
 
-You can see available updates in the `HostedCluster.Status` field of the `HostedCluster` custom resource (CR).
+You can update your hosted cluster by setting a channel that matches with your desired channel semantics.
 
-The available updates are not fetched from the Cluster Version Operator (CVO) of a hosted cluster. The list of the available updates can be different from the available updates from the following fields of the `HostedCluster` custom resource (CR):
+The initial `HostedCluster` resource does not have any information in the `status.version.availableUpdates` and `status.version.conditionalUpdates` fields. After you set the `spec.release` field to the {product-title} release version, the HyperShift Operator updates the `status.version` field with the available and conditional updates.
```

Comment:
Why did you change `spec.channel` here to `spec.release`?  [`spec.release`][1] is for requesting updates, but [`spec.channel`][2] is for configuring update advice.

[1]: https://github.com/openshift/hypershift/blob/b3b191f01e836a6496d5cafdee16ba24fe71f9f8/api/hypershift/v1beta1/hostedcluster_types.go#L437-L445
[2]: https://github.com/openshift/hypershift/blob/b3b191f01e836a6496d5cafdee16ba24fe71f9f8/api/hypershift/v1beta1/hostedcluster_types.go#L491-L497